lang/cs/Org.Apache.REEF.Client/AzureBatch/Service/AzureBatchService.cs

```diff
@@ -39,21 +40,39 @@ public sealed class AzureBatchService : IDisposable
         public string PoolId { get; private set; }
 
         private BatchClient Client { get; set; }
+        private ContainerRegistryProvider ContainerRegistryProvider { get; set; }
+        private IList<string> Ports { get; set; }
+        private ICommandBuilder CommandBuilder { get; set; }
+        private bool AreContainersEnabled
+        {
+            get
+            {
+                return this.ContainerRegistryProvider.IsValid();
+            }
+        }
+
         private bool disposed;
 
         [Inject]
         public AzureBatchService(
+            ContainerRegistryProvider containerRegistryProvider,
+            ICommandBuilder commandBuilder,
             [Parameter(typeof(AzureBatchAccountUri))] string azureBatchAccountUri,
             [Parameter(typeof(AzureBatchAccountName))] string azureBatchAccountName,
             [Parameter(typeof(AzureBatchAccountKey))] string azureBatchAccountKey,
-            [Parameter(typeof(AzureBatchPoolId))] string azureBatchPoolId)
+            [Parameter(typeof(AzureBatchPoolId))] string azureBatchPoolId,
+            [Parameter(typeof(AzureBatchPoolDriverPortsList))] IList<string> ports)
         {
-            BatchSharedKeyCredential credentials = new BatchSharedKeyCredential(azureBatchAccountUri, azureBatchAccountName, azureBatchAccountKey);
+            BatchSharedKeyCredential credentials = 
```

Comment:
make sure to trim trailing spaces, extra blank lines, etc. Use an autoformatting tool like CodeMaid

Reply:
The iteration #19 has cleanup using codemaid.

---
In reply to: [203900283](https://github.com/apache/reef/pull/1478#discussion_r203900283) [](ancestors = 203900283)